VaticanBAV/et/BAVet196.xml

```diff
@@ -0,0 +1,308 @@
+                          </decoNote>
+                        </decoDesc>
+
+                        <additions>
+                          <list>
+
+                            <item xml:id="a1">
+                              <locus target="#1v" />
+                              <desc type="Supplication">
+                                <title>ʾAsmāt Prayer</title>
+                              </desc>
+                              <q xml:lang="gez">
+                                <seg part="I">ትዮር፡ አላትዮር፡ ክትዮር፡ ፍክትዮር፡ በኃይለ፡ ዝንቱ፡ አስማቲከ፡ አንህዮ፡ ወአውርዶ፡ ለዝንቱ፡ በኖ፡ ዘሀሎ፡ ውስተ፡ ከርሱ፡ ለገብርከ፡ እገሌ፡</seg>
+                              </q>
+                            </item>
+
+                            <item xml:id="e1">
+                              <locus from="1r" to="2v" />
+                              <desc>Some scribbles in the margins in pen and pencil.</desc>
+                            </item>
+
+                          </list>
+                        </additions>
+
+                      </physDesc>
+
+                    </msPart>
+
+                    <msPart xml:id="p2">
+                      <msIdentifier><idno/></msIdentifier>
+
+                      <msContents>
+                        <summary/>
+
+                        <msItem xml:id="p2_i1">
+                          <locus from="55r" to="77v" />
+                          <title type="complete" ref="LIT2866RepCh147" />
+                          <textLang mainLang="gez" />
+                        </msItem>
+
+                        <msItem xml:id="p2_i2">
+                          <locus from="78r" to="80r" />
+                          <title>ʾAsmāt Prayer</title>
+                          <textLang mainLang="gez" />
+                          <incipit xml:lang="gez"><gap reason="ellipsis" extent="unknown" resp="PRS4805Grebaut PRS9530Tisseran" /> እስር፡ ጕርዔሆሙ፡ ለፀርየ፡ <gap reason="ellipsis" extent="unknown" resp="PRS4805Grebaut PRS9530Tisseran" /> በስመ፡ አብ፡ በል፡ ጸለየ፡ ምስለ፡ ዳዊት፡ ሳዶር፡ አላዶር፡ ዳናት፡ አዴራ፡ ሮዳስ፡</incipit>
+                          <note>Written in pencil.</note>
+                        </msItem>
+
+                        <msItem xml:id="p2_i3">
+                          <locus target="#80v" />
+                          <title/>
+                          <textLang mainLang="gez" />
+                          <incipit xml:lang="gez">ሀገርነ፡ ኮነት፡ ከመ፡ ሰማርያ፡ እንዘ፡ በላዕሌሃ፡ ወረደ፡ ነጠብጣበ፡ ዝናመ፡ መዓት፡ ወብሔር፡ ኃዘን፡ <gap reason="ellipsis" extent="unknown" resp="PRS4805Grebaut PRS9530Tisseran" /> አምጣነ፡ ረሐብ፡ ድምፀ፡ ዶርሖ፡ ያነቅሐነ፡ በሌሊት።</incipit>
+                        </msItem>
+                      </msContents>
+
+                      <physDesc>
+                        <objectDesc form="Codex">
+                          <supportDesc>
+                            <support>
+                              <material key="parchment" />
+                            </support>
+                            <extent>
+                              <measure unit="leaf">26</measure>
```

Comment:
Could you add here and in the first msPart (if it is known) a locus with the extent of the msPart in the manuscript?